topeft/modules/datacard_tools.py

```diff
@@ -787,7 +788,15 @@ def analyze(self,km_dist,ch,selected_wcs, crop_negative_bins):
         outf_root_name = self.FNAME_TEMPLATE.format(cat=ch,kmvar=km_dist,ext="root")
 
         h = self.hists[km_dist]
+        h_sumw2 = None
+        if f"{km_dist}_sumw2" in self.hists:
+            h_sumw2 = self.hists[km_dist+"_sumw2"]
+        else:
+            msg = "No sumw2 histogram found! Setting errors to 0"
+            print(msg)
+            h_sumw2 = None
```

Comment:
This last line `h_sumw2 = None` is redundant, but doesn't hurt leaving it in. So fine to keep it or drop it.

Reply:
Removed